modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1047,6 +1051,15 @@ public AnchorPoint anchorPoint() {
         }
         return sf.anchorPoint(ff.literal(anchor.getX()), ff.literal(anchor.getY()));
     }
+    
+    /**
+     * 
+     * @return True if the layer has a text-max-angle explicitly provided.
+     */
+    public boolean hasTextMaxAngle() throws MBFormatException {
+        return parse.isPropertyDefined(layout, "text-max-angle");
+    }
```

Comment:
I moved has `TextMaxAngle()` below the other textMaxAngle methods, it was above them before.  As far as public vs. private or protected, I was following suit for the surrounding code, which all appear to be public as well.